app/views/admin/words/new.html.erb

```diff
@@ -0,0 +1,46 @@
+
+<div class="box5">
+	<h1 class="text-center">Add Word</h1>
+		<%= form_for [:admin, @word], url: admin_category_words_path do |f| %>
```

Comment:
@YuheiTsunoda Pass `@category` information from controller, then move to partial file, So you can use same for edit page